src/routes/safe/components/Transactions/TxList/TxInfoCreation.tsx

```diff
@@ -21,7 +26,7 @@ export const TxInfoCreation = ({ transaction }: { transaction: Transaction }): R
   return (
     <TxDetailsContainer>
       <div className="tx-summary">
-        <div className="tx-hash">
+        <StyledGridRow>
```

Comment:
Why remove the class names?

Reply:
In this file (which is only for the Creation Tx) I only removed the classes that were not being targeted or that were not relevant for the new design.

Reply:
Don't forget E2E tests...